### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by hyk1st** [criyle/go-judge#54] [Q&A] [answered]
你好，我想问一下copyoutmax会影响到copyoutcache的大小吗

**Comment by criyle** [comment, marked as answer]
会

**Reply by hyk1st** [reply]
你好，我还想问一下copyout怎么让stdout不输出

**Reply by hyk1st** [reply]
就直接把stdout输出到copyoutcache，然后不输出到copycout

**Reply by hyk1st** [reply]
解决了哈哈哈，谢谢大佬

**Reply by criyle** [reply]
在 `copyOutCached` 列表中加入 `stdout`
